ArithmeticToolsTests/IsPowerOfTwoTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  IsPowerOfTwoTests.swift
+//  ArithmeticTools
+//
+//  Created by Brian Heim on 12/11/16.
+//  Copyright © 2016 James Bean. All rights reserved.
+//
+
+import XCTest
+@testable import ArithmeticTools
+
+class IsPowerOfTwoTests: XCTestCase {
```

Comment:
agree. i chose this name because there's IsDivisibleTests and to avoid confusion with the tests on the actual Power* classes.